### DISCUSSION TITLE
Query about the arguments in validation set section for DeePMD v2

### DISCUSSION THREAD

**Discussion by rajnichahal** [deepmodeling/deepmd-kit#2273] [General]
Hi, I have some confusion about the argument 'numb_btch' in the validation set section of training file for deepmd v2.

The documentation (https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html#training-validation-data) says that 'numb_btch'=An integer that specifies the number of systems to be sampled for each validation period. 
Does this mean that to include frames from each system in the validation, I should set 'numb_btch'=no. of systems I have in the validation section?

I believe that in v1 of deepmd, only 'numb_test' value needed to be specified. There I assume that as last set of each system was treated as the validation data, 'numb_test'= 100 (say) would be chosen randomly from all the systems to evaluate the test errors.

Please clarify. Thanks.

**Comment by njzjz** [comment]
Hi, the documentation was just fixed in #2257. See also #2231.

**Reply by rajnichahal** [reply]
Thank you!
